### ISSUE TITLE
给曲线上值最小的位置打点

### ISSUE THREAD

**Issue by zhpmatrix** [zhpmatrix/pac2017-final#5]
def saveFig(train_summary, val_summary, max_epoch, activity):
    train_loss = np.array(train_summary.read_scalar("Loss"))
    val_loss = np.array(val_summary.read_scalar("Loss"))
    
    plt.plot(train_loss[:,0], train_loss[:,1], label="curve_train_loss", color="red")
    
    min_idx = np.argmin(val_loss[:,1])
    plt.plot(min_idx, train_loss[min_idx][1], 'kx')
    show_min='['+str(min_idx) + ',' + str( float('%.3f' % val_loss[min_idx][1]) ) + ']'
    plt.annotate(show_min, xytext=(min_idx, val_loss[min_idx][1]), xy=(min_idx, val_loss[min_idx][1]))
    
    plt.plot(val_loss[:,0], val_loss[:,1], label="curve_val_loss", color='yellow')
    plt.scatter(val_loss[:,0], val_loss[:,1], label="scatter_val_loss", color='green')
    plt.title("Results")
    plt.legend()
    plt.savefig('../imgs/'+str(activity)+'-'+str(max_epoch)+'-results.jpg')
    plt.close()

**Comment by zhpmatrix** [comment]
打点所在的位置，会出现坐标不一致的情况！
